Tests/Integration/DSCResources/helper.psm1

```diff
@@ -101,3 +134,139 @@ function Get-ResourceMatchStatement
         }
     }
 }
+
+<#
+    .SYNOPSIS
+        Removes the rules that have dscresource equal to 'None'.
+        The integration tests randomly picks rules and if the rule has a dscresource='None' the test will fail
+        causing an intermitent build failure.
+
+    .PARAMETER Xml
+        The xml that will be examined.
+#>
+function Remove-DscResourceEqualsNone
+{
+    [CmdletBinding()]
+    [OutputType([xml])]
+    param
+    (
+        [Parameter(Mandatory = $true, ValueFromPipeLine = $true)]
+        [xml]
+        $Xml
+    )
+
+    $stigRuleNames = $Xml.DISASTIG | Get-Member -Type Property |
+        Where-Object -FilterScript {$PSItem.Name -match 'Rule$' -and $PSItem.Name -notmatch 'DocumentRule|ManualRule'}
+
+    # remove all dscresource -eq None
+    foreach ($stigRuleName in $stigRuleNames.Name)
+    {
+        foreach ($node in $Xml.DISASTIG.$stigRuleName.Rule)
+        {
+            if ($node.dscresource -eq 'None')
+            {
+                [void]$xml.DISASTIG.$stigRuleName.RemoveChild($node)
+            }
+
+        }
+    }
+
```

Comment:
Unnecessary line.

Reply:
So technically the style guidelines calls for two new lines after a closing curly unless followed by a closing curly brace:
https://github.com/PowerShell/DscResources/blob/master/StyleGuidelines.md#two-newlines-after-closing-brace